dynawo/sources/Models/CPP/ModelNetwork/DYNModelLine.cpp

```diff
@@ -507,69 +536,237 @@ ModelLine::evalNodeInjection() {
         modelBus2_->iiAdd(iiAdd2);
       }
     }
+  } else if (getConnectionState() == CLOSED) {
+    if (modelBus1_) {
+      double ur1Val = ur1();
+      double ui1Val = ui1();
+      double urp1Val = urp1();
+      double uip1Val = uip1();
+      double irAdd1 = conduct1_ * ur1Val + suscept1_ * urp1Val - suscept1_ * omegaRef_ * ui1Val + y_[IbReNum_];
```

Comment:
Should we have an omegaNom_ somewhere like for the RL equations ?

Reply:
Done